app/views/companies/_companies.html.erb

```diff
@@ -3,22 +3,25 @@
       <tr>
         <th scope="col">#</th>
         <th scope="col">Joint Stock Company</th>
-        <th scope="col">Symbol</th>
+        <th scope="col">Symbol/ISIN</th>
         <th scope="col">Stock</th>
         <th scope="col">Shares amount</th>
         <th scope="col">Free float</th>
       </tr>
     </thead>
     <tbody>
+      <% index = 0 %>
       <% companies.each do |company| %>
-        <tr>
-          <td class="text-dark" ><%= company.id %></td>
-          <td><%= link_to company.name, company, class: "nav-link text-dark p-0" %></td>
-          <td class="text-muted" ><%= company.isin %></td>
-          <td class="text-muted" >###</td>
-          <td class="text-muted" >###</td>
-          <td class="text-muted" >## %</td>
-        </tr>
+        <% if company.condition == '1' %>
```

Comment:
What does the condition `1` mean?

It should be easy to know looking at this code but it is not.

Perhaps you need a method on company, for example:

```ruby
company.in_good_condition?
# or
company.in_bad_condition?
end

Reply:
Maybe I can change name to `visible` , because `condition` is give kind of state, it shows if this object in db is company or something else for example kind of futures or another product and just determine if object is displayed or not.
Or maybe I can change value of `condition` to `visible` because in the future, there maybe more `condition'  values, like for example `old` which can say that company is not in the market now, or `suspended`

Reply:
I set `condition` for companies that actually should de display as `company active` , I think it is more easy to understand now what this attribute mean